spec/mongo/server_spec.rb

```diff
@@ -177,6 +177,14 @@
         expect(server.monitor.instance_variable_get('@thread')).to be nil
       end
     end
+  end
+
+  describe '#scan!' do
+    include_context 'with monitoring io'
+
+    let(:server_options) do
```

Comment:
Would the test still pass if this is  moved into the `with monitoring io' context below?